code/modules/vending/security.dm

```diff
@@ -13,12 +13,14 @@
 					/obj/item/reagent_containers/food/snacks/donut = 12,
 					/obj/item/storage/box/evidence = 6,
 					/obj/item/flashlight/seclite = 4,
+					/obj/item/holosign_creator/security = 3,
 					/obj/item/restraints/legcuffs/bola/energy = 7)
 	contraband = list(/obj/item/clothing/glasses/sunglasses/advanced = 2,
 					  /obj/item/storage/fancy/donut_box = 2)
 	premium = list(/obj/item/storage/belt/security/webbing = 5,
 					/obj/item/storage/backpack/duffelbag/sec/deputy = 4,
 				   /obj/item/coin/antagtoken = 1,
+				   /obj/item/grenade/barrier = 4,
```

Comment:
Aren't barrier grenades normally stashed in armory?

Reply:
Yes, which is odd, because they usually go underused. This is what incentivized me to make this change.

Maybe we get to see some proper use.

Reply:
meh, okay.